packages/xo-web/src/common/intl/messages.js

```diff
@@ -771,14 +771,15 @@ const messages = {
   addSrLabel: 'Add SR',
   addVmLabel: 'Add VM',
   addHostLabel: 'Add Host',
```

Comment:
Is it still in used?

Reply:
No, we keep it in order to not lose translation.

Reply:
Why? :thinking:

Reply:
I delet it. 
Not need to keep translation if we don't use it.
It's my fault.